src/examples/src/widgets/range-slider/Basic.tsx

```diff
@@ -1,8 +1,20 @@
 import { create, tsx } from '@dojo/framework/core/vdom';
+import icache from '@dojo/framework/core/middleware/icache';
 import RangeSlider from '@dojo/widgets/range-slider';
 
-const factory = create();
+const factory = create({ icache });
 
-export default factory(function Basic() {
-	return <RangeSlider />;
+export default factory(function Basic({ middleware: { icache } }) {
+	const min = 0;
+	const max = 100;
+	const value = icache.getOrSet('value', { min, max });
+
+	return (
+		<RangeSlider
+			value={value}
+			onValue={(newValue) => {
```

Comment:
is the range-slider not 'uncontrolled' yet with regards to it's value / onValue usage?